app/scripts/cartodb/categories-service.js

```diff
@@ -0,0 +1,79 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Categories ($log, $q, Config) {
+
+        var categories = null;
+        var categoriesQuery = [
+            'SELECT COUNT(*), SECTION, project_category, sub_category FROM master_datalist',
```

Comment:
Nit to pick: `SECTION` should be lower case in the query (column name, not SQL keyword).